CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+- Keep A Changelog files will now get a release header added automatically. This can be prevented with the `--no-release` argument. #TINY-9204
```

Comment:
Is the argument `--no-release` or `--no-changelog`? Looking at Parser.ts it looks like the latter assuming I am understanding it correctly.

Reply:
Oops, yeah it should be. I'm not sure how I got that wrong, but thanks anyways.